src/NzbDrone.Core.Test/IndexerTests/IntegrationTests/IndexerIntegrationTests.cs

```diff
@@ -16,26 +16,16 @@ namespace NzbDrone.Core.Test.IndexerTests.IntegrationTests
     [IntegrationTest]
     public class IndexerIntegrationTests : CoreTest
     {
-        private SingleEpisodeSearchCriteria _singleSearchCriteria;
-        private AnimeEpisodeSearchCriteria _animeSearchCriteria;
+        private AlbumSearchCriteria _singleSearchCriteria;
 
         [SetUp]
         public void SetUp()
         {
             UseRealHttp();
 
-            _singleSearchCriteria = new SingleEpisodeSearchCriteria()
+            _singleSearchCriteria = new AlbumSearchCriteria()
```

Comment:
Should we rename this `_albumSearchCriteria`?

Reply:
Probably a good idea